packages/hoprd/src/index.ts

```diff
@@ -279,59 +280,70 @@ async function main() {
   // 2. Create node instance
   try {
     logs.log('Creating HOPR Node')
-    node = createHoprNode(peerId, options)
+    node = createHoprNode(peerId, options, false)
```

Comment:
Side note, I believe boolean parameters should either be named options, or the function should have two variants to distinguish blatantly what the switch means. As is, one has to delve into the implementation to find out.

Reply:
I actually wanted to do something like `automaticStart = false` but TypeScript ain't fun. The alternative was passing an object that had that property which seemed a bit bloated, but can do.

Reply:
Resolving as I don't see this as a blocker, and otherwise, we should have a pattern as a team on how to handle this (e.g. all parameters are objects and thus options named, all parameters are passed as named variables, etc).